### PR TITLE
Created login page

### PR DESCRIPTION
**Pull request by edenmaor**
Added a login page that extends the home page
Created a views.py inside side_hustle

**please review code** 

Closed PR #31 
#41 Assigned by shay

### CODE REVIEW

**Review by assafg95 — APPROVED** [review]

File: side_hustle/urls.py [hunk 1]
```diff
@@ -15,7 +15,11 @@
 """
 from django.contrib import admin
 from django.urls import path
+from .views import login, home
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('login/', login),
+    path('', home, name='home')
```
Comment (AshkenYariv) [inline side_hustle/urls.py]: where does this home lead regarding the login page?

**Review by shayakrach — CHANGES_REQUESTED** [review]
Please squeeze all the commits.

File: templates/home.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+    </div>
+   </div>
+  </div>
+</div>
+</div>
+</div>
+</body>
+</html>
```
Comment (AshkenYariv) [inline templates/home.html:136]: The HTML page looks amazingggg!!

**Review by AshkenYariv — COMMENTED** [review]
Just one comment that I left, in the url do we need the home that you wrote, after that is resolved I approve.

File: side_hustle/urls.py (hunk 1 above)
Comment (edenmaor) [reply side_hustle/urls.py]: in terms of django, it extends the home page and allow us later add and redirect to other pages from the home. a more hierarchal approach.

**Pull request closed without merging**